### ISSUE TITLE
Building issues

### ISSUE THREAD

**Issue by timoshimmo** [unhurdle/spectrum-royale#74]
i can't seem to be able to build the project successfully. I always get the error "The library C:\..........\spectrum-royale-main\Spectrum\target\Spectrum.swc could not be found". Pls help

**Comment by Harbs** [comment]
You need to first build Spectrum.swc:
1. `cd` into the Spectrum folder
2. Run `npm install`
3. Run either `ant` or `asconfigc` (or just build in VS Code).
4. `cd ../SpectrumBrowser`
5. You should be able to build the Browser app.

**Comment by timoshimmo** [comment]
Yes i did that... After npm install, i see a playerglobal.swc file in the Spectrum folder. I then run asconfigc. I just get the same error. Also based on the asconfig.json, the output doesn't seem to create a target/Spectrum.swc.

**Comment by Harbs** [comment]
Which folder are you running it in? Make sure you first run it in the Spectrum sub-folder.

**Comment by timoshimmo** [comment]
Yes I'm running it in the Spectrum subfolder. I'm running it in spectrum-royale-master/Spectrum folder. The same folder I run the npm install and the build.

**Comment by Harbs** [comment]
What error do you get in the when run in the Spectrum folder? That should be creating the swc unless the swf build fails.

If the swf build fails you should be getting another error further up.

**Comment by timoshimmo** [comment]
![spectrum-royale-build-error](https://user-images.githubusercontent.com/2452477/147891842-694a3a03-5120-4f41-b2cd-67eeb98c3794.png)

This is the error i'm getting when i build using asconfigc in the Spectrum subfolder.

**Comment by Harbs** [comment]
OK. You are right. I think I've always built using asconfigc after a successful build with ant. I'll commit a fix soon, but it will only work with the latest nightly build of Royale because I'm using a function added yesterday. You can backport the changes if you don't want to use the latest nightly.

**Comment by Harbs** [comment]
Thanks for reporting this!

**Comment by Harbs** [comment]
The changes in 07d26d6b7f077237b22d83c85bc683e18e683b31 will fix this.

**Comment by Harbs** [comment]
Didn't fix it. For now I recommend building with `ant`. When I have time, I'll see if I can fix the `asconfigc` build.

**Comment by timoshimmo** [comment]
Okay thanks... I'll focus on building it with ant.
